h2o/src/main/java/org/apache/mahout/h2obindings/drm/H2OBCast.java

```diff
@@ -118,4 +118,16 @@ private T deserialize(byte buf[]) {
     }
     return ret;
   }
+
+  /**
+   * Stop broadcasting when called on driver side. Release any network resources.
+   *
+   */
+  @Override
+  public void close() throws IOException {
+
+    // TODO: review this. It looks like it is not really a broadcast mechanism but rather just a
```

Comment:
Yes, h2o's broadcast is just serialize + multiple unicast (almost like spark's http based broadcast)